code/src/css/detail.css

```diff
@@ -0,0 +1,101 @@
+.details-container {
+  display: inline-block;
+  background-size: cover;
+  background-position: center;
+  position: absolute;
+  overflow: hidden;
+  height: 100%;
+  width: 100%;
+  background: linear-gradient(190deg, rgba(0,0,0,0) 60%, rgba(0,0,0,1) 100%);
+  font-weight: 700;
+}
+.movie-background {
+  display:inline-block;
+  position: fixed;
+  z-index: -1;
+  background-size: cover;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  background-repeat: no-repeat ;
+}
+ .detailGrid {
+  display: grid;
+  overflow: scroll;
+}
+button {
+  background-color: red;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  font-variant: small-caps;
+  font-size: 24px;
+  line-height: 0.8;
+  margin-top: 2%;
+  margin-left: 2%;
+  padding: 10px;
+  border-style: none;
+  height: 3em;
+  border-radius: 10px 40px;
+  width: 120px;
+  height: 80px;
+  cursor: pointer;
+}
+button:hover {
+  box-shadow: 0 5px 5px 0 rgba(0, 0, 0, 0.24), 0 17px 50px 0 rgba(0, 0, 0, 0.19);
```

Comment:
This button is absolutley amazing, love it! Awesome job here with the styling 🤩 !! 
Good job with the media quierys with the grid etc! 

On small thing in the CSS file is that you could have some space between the different targets to make it easier to read.